File: Palindrome/Palindrome.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        return str(x)[::-1] == str(x)


#! SOLUTION 2
# class Solution:
#     def isPalindrome(self, x: int) -> bool:
#         xx = str(x);
#         ll = int(len(xx)/2);

#         for i in range(ll):
#             if(xx[i] != xx[-i-1]):
#                 return False;
#         return True;

File: Palindrome/Palindrome.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    
    let text = x.toString();
    for(var i=0;i<text.length/2;i++)
    {
        if(text.charAt(i) != text.charAt(text.length-1-i))
        {
            return false;
        }

    }
    return true;
};